docs/ja/index.rst

```diff
@@ -23,7 +23,10 @@ CakePHPから `composer <https://getcomposer.org/>`_ でプラグインをイン
 
 認証プラグインは、ミドルウェアとしてアプリケーションと統合します。 `middleware <http://book.cakephp.org/4/en/controllers/middleware.html>`_
 また、認証されていないアクセスをより簡単にするためのコンポーネントとして使用することもできます。  まずはミドルウェアを適用してみましょう。
- **src/Application.php** に以下のクラスを追加します。
+
+**src/Application.php** に以下のクラスを追加します。
+=======
```

Comment:
Is this supposed to be a heading or are these merge conflict markers?

Reply:
It is a magic conflict marker.
It would be great if you could delete the old one and get rid of that too.
Sorry for not checking.

Reply:
I can't easily push changes to this branch. Is there a way you could clean this up?

Reply:
Yes. I'll try.